index.js

```diff
@@ -83,6 +113,7 @@ module.exports = (text, options) => {
 		dimBorder: false,
 		align: 'left',
 		float: 'left',
+		alignTitle: 'left',
```

Comment:
Should this be `titleAlign` ?

Reply:
I honestly don't have a preference between those two.

I'm unavailable for whole of July, that's why this thread is in pause right now.

Reply:
I think it should be `titleAlignment`